external/Kconfig

```diff
@@ -22,13 +22,6 @@ config LWM2M_SERVER_MODE
 		enable LWM2M WAKAAMA server mode
 endchoice
 
-config EXTERNAL_WAKAAMA_EXAMPLE
-	bool "EXTERNAL_WAKAAMA_EXAMPLE"
-	depends on LWM2M_CLIENT_MODE || LWM2M_SERVER_MODE
-	default n
-	---help---
-		enable LWM2M WAKAAMA server or client example
-
```

Comment:
Dear. dr-venkman

Sorry for late response.
We're preparing next pull request including separation of example for DS SW R&D center
all wakaama codes will be reverted to remove DS SW R&D center changes

Thanks